rollup.config.js

```diff
@@ -19,8 +23,41 @@ const processEnv = conf => ({
   'process.env': `(${JSON.stringify(conf)})`,
 });
 
-const vuePlugin = process.env.VUE_VERSION === 'vue3' ? vueV3 : vueV2;
-const outputDir = process.env.VUE_VERSION === 'vue3' ? 'vue3' : 'vue2';
+const vuePlugin = isVue3 ? vueV3 : vueV2;
+const outputDir = isVue3 ? 'vue3' : 'vue2';
+
+const excludeCompositionsAPI = {
+  load(id) {
+    if (id.endsWith('/src/compositions/index.js')) {
+      return '';
```

Comment:
Import with `'/src/compositions/index.js'` gets replaced with an empty string. It's to exclude this line from Vue 2 bundle:
https://github.com/algolia/vue-instantsearch/blob/58618c5609c17e8feb3f842dfa2a30b2cb72721b/src/instantsearch.js#L7